examples/lighting-app/lighting-common/color_format/color_format.cpp

```diff
@@ -0,0 +1,126 @@
+/*

```

Comment:
Probably OK, though still seems like this should be in the SDK, not in the example app...